### PR TITLE
Svgcreator into object

### PR DESCRIPTION
**Pull request by agudek**
Changed svgCreator into object and constructed it globally.
svg creator functions can be called with svgCreator.<functionName>()

### CODE REVIEW

File: src/svgCreator.js [hunk 1]
```diff
@@ -1,225 +1,228 @@
...
+        }
+        return g;
+    }
+
+    function createXAxisLabel(svg, module) {
+        var label = "";
+        if((label = octopeerHelper.getSafeModuleValue(module,"xAxisLabel")) !== "") {
+        var g = svg.append("g");
+            
+            //var rect = g.append("rect");
```
Comment (borek2) [inline src/svgCreator.js]: Removed unused comments ! :)

File: src/svgCreator.js [hunk 2]
```diff
@@ -1,228 +1,214 @@
-/* exported createSVG */
+/* exported SvgCreator */
 /*globals octopeerHelper*/
 /*jshint esnext: true */
 /* jshint unused : vars */
+function SvgCreator() {
+
+    function createSvgContainer(w = 720, h = 350) {
```
Comment (mboom) [inline src/svgCreator.js]: Maybe we should define width and height as global variables. I have seen these vars on multiple places.

File: src/svgCreator.js [hunk 3]
```diff
@@ -1,228 +1,214 @@
...
+                .attr("y", 0)
+                .attr("x", 9)
+                .attr("dy", ".35em")
+                .attr("transform", "rotate("+octopeerHelper.getSafeModuleValue(module,"yAxisLabelRotation")+")")
+                .style("text-anchor", "start");
+        }
+
+        if(octopeerHelper.getSafeModuleValue(module,"yAxisLine")) {
+        	axisContainer.append("line")
+                .attr("y1",350-50-10)
```
Comment (mboom) [inline src/svgCreator.js]: Also these number: 350, 50 10 are hardcoded multiple times. Same for line 67 and 70 and many more places. Maybe define these numbers as global vars?

**Comment by mboom** [comment]
Nice structure, just a couple of minor comments. Maybe they should be improved in a new pull-request, because the issue appears in the whole repository.

**Comment by agudek** [comment]
Good idea! Will address that in the following sprint.

**Pull request merged**